mflowgen/common/rtl/gen_rtl.sh

```diff
@@ -3,34 +3,74 @@ if [ $soc_only = True ]; then
   echo "soc_only set to true. Garnet not included"
   touch outputs/design.v
 else
-  current_dir=$(pwd)
-  cd $GARNET_HOME
-  if [ -d "genesis_verif/" ]; then
-    rm -rf genesis_verif
+  # Clean out old rtl outputs
+  if [ -d "$GARNET_HOME/genesis_verif/" ]; then
+    rm -rf $GARNET_HOME/genesis_verif
```

Comment:
probably can do this without the check, but nbd.

Reply:
done